common/pop_types.txt

```diff
@@ -120,7 +120,7 @@ promotion_chance = {
 	modifier = {
 		factor = -100
 		pop_majority_ideology = tribal_ideology
-		country = { civilized = no }
+		NOT = { culture = this }
```

Comment:
Isn't the idea to have primary and accepted tribal pops promote/demote out of being tribals? `culture = THIS` returns true when the province the pop is in is majority primary culture